lib/generators/install/templates/microservices_engine.rb

```diff
@@ -0,0 +1,22 @@
+require 'net/http'
+config_file = '../mse_router_info.yml'
+puts("[MSE] > No router configuration YAML file found") and return if
```

Comment:
It does, but it is an exceptional circumstance which means we can't initialize the Rails app, so I think it's appropriate.